<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camino Más Corto</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="loadButton">
      <property name="text">
       <string>Cargar Grafo</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Inicio:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="startSpinBox">
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Fin:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="endSpinBox">
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="calculateButton">
      <property name="text">
       <string>Calcular Camino Más Corto</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="resultTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
